samples/matter/common/src/thread_util.cpp

```diff
@@ -11,8 +11,21 @@
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/internal/DeviceNetworkInfo.h>
 
+#ifdef CONFIG_MPSL_FEM
+#include <openthread/platform/radio.h>
+#include <platform/OpenThread/OpenThreadUtils.h>
+#endif
+
 #include <zephyr/zephyr.h>
 
+#ifdef CONFIG_MPSL_FEM
+#ifndef CONFIG_FEM_802_15_4_DEFAULT_TX_POWER
+#define CONFIG_FEM_802_15_4_DEFAULT_TX_POWER 20
+#endif
+constexpr static int8_t kMinThreadOutputPower = -40;
```

Comment:
These can be defined at the Kconfig level using `range` keyword, so that no runtime checks are needed.